hooks/useYouTubeChannel.ts

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+
+const useYouTubeChannel = (
+  artistAccountId: string | undefined,
+  accountId: string | undefined
```

Comment:
both of these params exist in a `provider` right?
- why are you passing these as input params rather than retrieving them directly from providers?